integration-tests/src/prover_api.rs

```diff
@@ -1,35 +1,69 @@
+use crate::dyn_wallet_provider::EthDynProvider;
+use crate::network::Zksync;
+use crate::provider::ZksyncApi;
+use alloy::primitives::{U256, keccak256};
+use alloy::providers::{DynProvider, Provider};
+use alloy::rpc::types::Filter;
 use backon::{ConstantBuilder, Retryable};
-use reqwest::{Client, StatusCode};
 use std::time::Duration;
+use zksync_os_l1_sender::l1_discovery::get_l1_state;
+
+/// Default bridgehub address from zkos-l1-state.json
+const DEFAULT_BRIDGEHUB_ADDRESS: &str = "0x70968ad336b957311e3c1c63e36d05035e356f68";
```

Comment:
No need to hardcode this - you can get bridgehub address by calling `l2_zk_provider.get_bridgehub_contract()`